client/babylon/game.jsx

```diff
@@ -142,17 +140,21 @@ function createImagePlane(type, sphere, scene) {
     material.emissiveTexture = webcamTexture;
     material.diffuseTexture = webcamTexture; // same texture must be assigned to both emissive and diffuse, diffuse only is too dark and emissive only is too bright and washed out
   }
+  // mesh.outlineColor = Color3.Red();
```

Comment:
will we need any of these commented out lines in the future or does lie 146 pretty much take care of them?